flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/SinkCodeGenerator.scala

```diff
@@ -223,10 +224,10 @@ object SinkCodeGenerator {
           case (fieldTypeInfo, i) =>
             val requestedTypeInfo = tt.getTypeAt(i)
             validateFieldType(requestedTypeInfo)
-            if (!areTypesCompatible(
-              fromTypeInfoToLogicalType(fieldTypeInfo),
-              fromTypeInfoToLogicalType(requestedTypeInfo)) &&
-              !requestedTypeInfo.isInstanceOf[GenericTypeInfo[Object]]) {
+            if (!PlannerTypeUtils.isAssignable(
+                fromTypeInfoToLogicalType(fieldTypeInfo),
```

Comment:
checkstyle?

Reply:
I added an additional indent to make the code readable (separate from the following code block). But I can revert it if you think the original style is correct.

Reply:
I think Scala only one indent is enough. Correct me if I wrong.

Reply:
OK.